examples/assets-and-custom-transfer-logic/README.md

```diff
@@ -0,0 +1,42 @@
+# Multisignature delegated account support
+
+This example demonstrates authorizing asset transfer transactions based on custom logic by smart contract.
+
+## Overview
+
+Using Algorand’s native layer one Assets, developers can create a token that represents either a real world or digital good, service, or resource in a matter of minutes. No smart contract is required. These assets are very powerful and allow for flexibility in how they are traded and controlled. Typically assets are traded as any other token on the chain. All that is required is to issue an `asset transfer transaction`.
+
+This is fine for most cases, but what if you want some custom logic to execute to approve the transfer. This may be the case when you have one of the following scenarios.
+
+- KYC/AML - The user’s identity must be verified before the transaction is approved
+- Extra Fees Required - Such as taxes, commission on real estate, or some basis point fee must be paid.
+
+This solution explains the process of how this can be done in Algorand using a Smart Contract Application.
+
+## Problem Solution
+
+Using Algorand’s Atomic Transaction feature allows multiple transactions to be submitted at one time and if any of the transactions fail, then they all fail. So, one way of solving this issue is to group an asset transfer transaction with a call to a stateful smart contract and submit them simultaneously. The only caveat to this is that nothing prevents the asset transfer transaction being submitted by itself.
```

Comment:
```suggestion
Using Algorand’s Atomic Transaction feature allows multiple transactions to be submitted at one time and if any of the transactions fail, then they all fail. So, one way of solving this issue is to group an asset transfer transaction with a call to a stateful smart contract and submit them simultaneously. The only limitation to this is that nothing prevents the asset transfer transaction being submitted by itself.
```

Reply:
can you please explain the limitation ? what do you mean by submitted by itself ?

Reply:
It means that the creator can execute the transaction (even if the asset is frozen). For this we set the set the `defaultFrozenAddress` to `true` and asset is transferred through a clawback address (which is actually an escrow account in this case). More details can be found here at the reference: [https://developer.algorand.org/solutions/assets-and-custom-transfer-logic/](https://developer.algorand.org/solutions/assets-and-custom-transfer-logic/)

Reply:
please add more clear description in readme also

Reply:
Added more description.

Reply:
Ok